website/docs/terms/elt.md

```diff
@@ -0,0 +1,139 @@
+
+Since not every data source will integrate with SaaS tools for extraction and loading, it’s sometimes inevitable that teams will write custom ingestion scripts in addition to their SaaS tools.
+
+### Load
+
+During the loading stage, data that was extracted is loaded into the target data warehouse. Some examples of modern data warehouses include Snowflake, Amazon Redshift, and Google BigQuery. Examples of other data storage platforms include data lakes such as Databricks’s Data Lakes. Most of the SaaS applications that extract data from your data sources will also load it into your target data warehouse. Custom or in-house extraction and load processes usually require strong data engineering and technical skills.
+
+At this point in the ELT process, the data is mostly unchanged from its point of extraction. If you use an extraction and loading tool like Fivetran, there may have been some light normalization on your data. But for all intents and purposes, the data loaded into your data warehouse at this stage is in its raw format.
+
+### Transform
+
+In the final transformation step, the raw data that has been loaded into your data warehouse is finally ready for modeling! When you first look at this data, you may notice a few things about it…
+
+- Column names may or may not be clear
+- Some columns are potentially the incorrect data type
+- Tables are not joined to other tables
+- Timestamps may be in the incorrect timezone for your reporting
+- JSON fields may need to be unnested
+- Tables may be missing primary keys
+- And more!
+
+...hence the need for transformation! During the transformation process, data from your data sources is usually:
+
+- **Lightly Transformed**: Fields are cast correctly, timestamp fields’ timezones are made uniform, tables and fields are renamed appropriately, and more. 
+- **Heavily Transformed**: Business logic is added, appropriate materializations are established, data is joined together, etc. 
+- **QA’d**: Data is tested according to business standards. In this step, data teams may ensure primary keys are unique, model relations match-up, column values are appropriate, and more.
+
+Common ways to transform your data include leveraging modern technologies such as dbt, writing custom SQL scripts that are automated by a scheduler, utilizing stored procedures, and more.
+
+## ELT vs ETL
+
+The primary difference between the traditional ETL and the modern ELT workflow is when data transformation and loading take place. In ETL workflows, data extracted from data sources is transformed prior to being loaded into target data platforms. Newer ELT workflows have data being transformed after being loaded into the data platform of choice. Why is this such a big deal?
+
+|  | ELT | ETL |
```

Comment:
Same as ETL page, think this is using the old version!

Reply:
Good catch! I totally missed the changes to the table. Fixed in the latest commit